96.md

```diff
@@ -84,46 +82,43 @@ it must use the "api_url" field instead.
 
 See https://github.com/aljazceru/awesome-nostr#nip-96-file-storage-servers.
 
+## Auth
+
+When indicated, `clients` must add an [NIP-98](98.md) `Authorization` header (**optionally** with the encoded `payload` tag set to the base64-encoded 256-bit SHA-256 hash of the file - not the hash of the whole request body).
+
 ## Upload
 
-A file can be uploaded one at a time to `https://your-file-server.example/custom-api-path` (route from `https://your-file-server.example/.well-known/nostr/nip96.json` "api_url" field) as `multipart/form-data` content type using `POST` method with the file object set to the `file` form data field.
+`POST $api_url` as `multipart/form-data`.
 
-`Clients` must add an [NIP-98](98.md) `Authorization` header (**optionally** with the encoded `payload` tag set to the base64-encoded 256-bit SHA-256 hash of the file - not the hash of the whole request body).
-If using an html form, use an `Authorization` form data field instead.
+**AUTH required**
 
-These following **optional** form data fields MAY be used by `servers` and SHOULD be sent by `clients`:
-- `expiration`: string of the UNIX timestamp in seconds. Empty string if file should be stored forever. The server isn't required to honor this;
-- `size`: string of the file byte size. This is just a value the server can use to reject early if the file size exceeds the server limits;
-- `alt`: (recommended) strict description text for visibility-impaired users;
-- `caption`: loose description;
-- `media_type`: "avatar" or "banner". Informs the server if the file will be used as an avatar or banner. If absent, the server will interpret it as a normal upload, without special treatment;
-- `content_type`: mime type such as "image/jpeg". This is just a value the server can use to reject early if the mime type isn't supported.
+List of form fields:
 
+- `file`: **REQUIRED** the file to upload
+- `caption`: **RECOMMENDED** loose description;
+- `expiration`: UNIX timestamp in seconds. Empty string if file should be stored forever. The server isn't required to honor this.
+- `size`: File byte size. This is just a value the server can use to reject early if the file size exceeds the server limits.
+- `alt`: **RECOMMENDED** strict description text for visibility-impaired users.
+- `media_type`: "avatar" or "banner". Informs the server if the file will be used as an avatar or banner. If absent, the server will interpret it as a normal upload, without special treatment.
+- `content_type`: mime type such as "image/jpeg". This is just a value the server can use to reject early if the mime type isn't supported.
 
```

Comment:
Suggested "no transformations" option:

+- `no_transform`: "true" asks server to do no transformations and serve the uploaded file as is, may be rejected. By default it's up to the server.